api/server/volume.go

```diff
@@ -1305,6 +1305,30 @@ func (vd *volAPI) usedsize(w http.ResponseWriter, r *http.Request) {
 	json.NewEncoder(w).Encode(used)
 }
 
+func (vd *volAPI) volumeUsageByNode(w http.ResponseWriter, r *http.Request) {
+	var nodeID string
+	var err error
+	if nodeID, err = vd.parseID(r); err != nil {
+		e := fmt.Errorf("Failed to parse nodeID: %s", err.Error())
+		http.Error(w, e.Error(), http.StatusBadRequest)
+		return
+	}
+
+	d, err := vd.getVolDriver(r)
+	if err != nil {
+		notFound(w, r)
+		return
+	}
+
+	used, err := d.VolumeUsageByNode(nodeID)
+	if err != nil {
+		e := fmt.Errorf("Failed to get VolumeUsageByNode: %s", err.Error())
```

Comment:
instead of wrapping the error, suggest passing it as it is.

Reply:
Done